### ISSUE TITLE
[BUILD]

### ISSUE THREAD

**Issue by PrimeTDMomega** [Wolfsurge/Paragon#46]

**Comment by SooStrator1136** [comment]
I think what you are looking for are the workflow artifacts? Check out the link in the readme if thats the case

**Comment by PrimeTDMomega** [comment]
![Wolfsurge_Paragon_ A 1 12 2 utility mod intended for anarchy servers - Opera 12_5_2022 7_46_23 PM](https://user-images.githubusercontent.com/105910256/205659044-fac4806d-013c-4278-8638-6146a7497031.png)
That is what I was asking you all to build.

**Comment by Wolfsurge** [comment]
Yes, click on "Download the client from the latest commit here" in the readme.

**Comment by SooStrator1136** [comment]
https://nightly.link/Wolfsurge/Paragon/workflows/build/master/Paragon-Build.zip

**Comment by PrimeTDMomega** [comment]
Thx.
I found it, the releases tab showing `1.0.0` was a bit misleading lol
